### PR TITLE
Added autocoverage support

### PR DESCRIPTION
**Pull request by ImmortalShox**
Hi! I'm adding autocoverage support for begiak, now with command ".awikstats coverage [here 3 letter language code]" you can run coverage and it will automatically update the wiki stats page after coverage is completed. There aren't many changes here, most changes are in scripts on SVN.

### CODE REVIEW

File: modules/apertium_wikistats.py [hunk 1]
```diff
@@ -47,10 +52,47 @@ def awikstats(phenny, input):
...
+                if out.startswith('Coverage:'):
+                    phenny.say('%s: %s [%s]' % (input.nick, out, lang))
+                else:
+                    for line in stderr.splitlines():
+                        phenny.msg(input.nick, line)
+            except:
+                for line in stderr.splitlines():
+                    phenny.msg(input.nick, line)
+        else:
+            phenny.say('%s: Sorry, there can be only one coverage running!' % input.nick)
```
Comment (sushain97) [inline modules/apertium_wikistats.py]: Might be nice to mention which one (i.e make the global a string instead of a book).

Comment (sushain97) [reply modules/apertium_wikistats.py]: *bool. Mobile won't let me edit comments...

Comment (ImmortalShox) [reply modules/apertium_wikistats.py]: screen shot 2017-12-27 at 17 46 06
screen shot 2017-12-27 at 17 31 28
screen shot 2017-12-27 at 17 53 55

Comment (sushain97) [reply modules/apertium_wikistats.py]: Looks good. Let's let @jonorthwash have the final say ;)

**Comment by shardulc** [comment]
@ImmortalShox When it reports the coverage, can you also make it report the link to the wiki page for convenience?

**Pull request merged**